infra-aws/README.md

```
@@ -0,0 +1,35 @@
+  access_key = "YOU_ACCESS_KEY"
+  secret_key = "YOU_SECRET_KEY"
+  }
+```
+3. `terraform init`
+4. `terrafrom apply`
+
+# Steps to update infra
+
+1. `cd ../`
+2.  Add access key and secret key to provider "aws"{} 
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)